test/schema/basalTest.js

```diff
@@ -242,7 +241,7 @@ describe('schema/basal.js', function(){
     });
 
     describe('scheduleName', function(){
-      helper.rejectIfAbsent(goodObject, 'scheduleName');
+      helper.okIfAbsent(goodObject, 'scheduleName');
```

Comment:
so these, `scheduleName `, are ok to be absent because of tandem or its all good in general ?